practice-app/routes/exchange_rate_api.js

```diff
@@ -38,6 +38,30 @@ router.get('/', (req, res) => {
     });
 });
 
+router.get('/avg', (req, res) => {
+    let from = req.query.from || 'TRY';
+    let to = req.query.to;
+    let start_date = req.query.start_date || moveDate('', -7);
+    let end_date = req.query.end_date || moveDate('', 0);
+
+    if (to == null) {
+        res.status(400).send({
+            'error': '\'to\' parameter cannot be empty!'
+        });
+        return;
+    }
+    var startDate = new Date(start_date); 
```

Comment:
just a minor comment, feel free to discard it:  you can use let or const instead of var for better variable scoping